appcues/src/main/java/com/appcues/debugger/screencapture/AndroidTargetingStrategy.kt

```diff
@@ -195,16 +210,17 @@ private fun View.extractResourceName(): String? {
     }
 }
 
-private fun SemanticsNode.asCaptureView(context: Context): ViewElement {
+private fun SemanticsNode.asCaptureView(context: Context, screenBounds: Rect): ViewElement? {
     val displayMetrics = context.resources.displayMetrics
     val density = displayMetrics.density
 
-    var childElements: List<ViewElement>? = children.map {
-        it.asCaptureView(context)
+    // if the view is not currently in the screenshot image (scrolled away), ignore
+    if (screenBounds.contains(unclippedGlobalBounds).not()) {
+        return null
```

Comment:
we were not previously checking Compose views to see if they were in the screen bounds - this fixes that portion. I plan to pull this out into a separate PR on main, as it should be fixed regardless of where this retry work ends up.

Reply:
oh I guess our check works for the first ComposeView, but the subsequent ones (inside the SemanticsNode) can be placed outside, so this extra check makes sense, so in this case should this be similar to isFullyVisible? (with tolerance maybe)